tests/foreman/cli/test_reporttemplates.py

```diff
@@ -63,878 +63,906 @@
+        2. A fake host to be checked in report output
 
-        :CaseImportance: Medium
+    :steps:
+        1. Check settings for default Ansible Inventory template name and ensure
+           the template is present
+        2. Try to render the template using the user with ATIR role
+        3. Check the fake host is present in the output
 
-        :BZ: 1750924
-        """
-        host_name = gen_string('alpha')
-        host = make_fake_host({'name': host_name})
+    :expectedresults: Report template is present, renederable and provides output
 
-        result = ReportTemplate.generate({'name': 'Host - Statuses', 'organization': DEFAULT_ORG})
+    :CaseImportance: Medium
+    """
+    settings = Settings.list({'search': 'name=ansible_inventory_template'})
+    assert 1 == len(settings)
+    template_name = settings[0]['value']
 
-        assert host['name'], [item.split(',')[0] for item in result]
+    report_list = ReportTemplate.list()
+    assert template_name in [rt['name'] for rt in report_list]
 
-    @pytest.mark.tier2
-    @pytest.mark.skip_if_open('BZ:1782807')
-    def test_positive_generate_ansible_template(self):
-        """Report template named 'Ansible Inventory' (default name is specified in settings)
-        must be present in Satellite 6.7 and later in order to provide enhanced functionality
-        for Ansible Tower inventory synchronization with Satellite.
+    login = gen_string('alpha').lower()
+    password = gen_string('alpha').lower()
+    loc = Location.info({'name': DEFAULT_LOC})
+    org = Org.info({'name': DEFAULT_ORG})
+
+    user = make_user(
+        {
+            'login': login,
+            'password': password,
+            'organization-ids': org['id'],
+            'location-ids': loc['id'],
+        }
+    )
+
+    User.add_role({'login': user['login'], 'role': 'Ansible Tower Inventory Reader'})
+
+    host_name = gen_string('alpha').lower()
+    host = make_fake_host({'name': host_name})
+
+    schedule = ReportTemplate.with_user(username=user['login'], password=password).schedule(
+        {'name': template_name}
+    )
+
+    report_data = ReportTemplate.with_user(username=user['login'], password=password).report_data(
+        {'id': template_name, 'job-id': schedule[0].split('Job ID: ', 1)[1]}
+    )
 
-        :id: f1f7adfc-9601-4498-95c8-3e82e2b36583
+    assert host['name'] in [item.split(',')[1] for item in report_data if len(item) > 0]
 
-        :setup:
-            1. A user with minimal required permissions: 'Ansible Tower Inventory Reader' role
-            2. A fake host to be checked in report output
 
-        :steps:
-            1. Check settings for default Ansible Inventory template name and ensure
-               the template is present
-            2. Try to render the template using the user with ATIR role
-            3. Check the fake host is present in the output
+@pytest.mark.tier3
+def test_positive_generate_entitlements_report_multiple_formats(
```

Comment:
Add `@pytest.mark.libvirt_content_host` marker for deselection !

Reply:
Good call, your PR went in after I opened this up, thanks!